tox.ini

```
@@ -1,5 +1,7 @@
 [tox]
-envlist = py35,py36,py37,py38,lint,check-formatting,manifest
+minversion = 3.4.0
+skip_missing_interpreters=true
```

What's the benefit of setting this to true? I'd want CI to fail if I can't test certain versions

---

Hmm I think the benefit is less with this set to true - I don't mind log spam as much as I want both local dev & CI to fail loudly if something is wrong.

---

Ok point taken, I will remove that option